src/tests/test_activities.py

```diff
@@ -7,6 +7,14 @@
 class ActivitiesTestCase(BaseTestCase):
     """Test activities endpoints."""
 
+    def setUp(self):
+        """Save required roles."""
+        BaseTestCase.setUp(self)
+        self.successops_role.save()
+        self.fellow_role.save()
+        self.success_role.save()
+        self.finance_role.save()
+
     def test_create_activity(self):
         """Test that an activity has been created successfully."""
         new_activity = dict(name="tech congress",
```

Comment:
Could you also change the hard-coded `activityDate`s in this file? Jonathan missed them and these tests will be useless in 2020.